deploy/helm/sumologic/conf/logs/logs.source.containers.conf

```diff
@@ -46,6 +46,11 @@
   <filter containers.**>
     @type kubernetes_sumologic
     @include logs.kubernetes.sumologic.filter.conf
+    {{ if eq .Values.sumologic.collectionMonitoring false}}
+    exclude_namespace_regex {{ .Release.Namespace | quote }}
+    {{- else}}
+    exclude_namespace_regex {{ .Values.fluentd.logs.containers.excludeNamespaceRegex | quote }}
```

Comment:
we'll want to do a combination of the two, we don't want to ignore the customer's `excludeNamespaceRegex` just because they want to stop collecting the `sumologic` data

Reply:
done

Reply:
since this is regex should it be `"%s|%s"` instead of comma separated?